internal/pkg/manifest/pipeline.go

```diff
@@ -187,8 +175,23 @@ func validateVersion(pm *pipelineManifest) (PipelineSchemaMajorVersion, error) {
 	}
 }
 
-// CFNTemplate serializes the manifest object into a CloudFormation template.
-func (m *pipelineManifest) CFNTemplate() (string, error) {
-	// TODO: #223 Generate CFN template for the archer pipeline
+func (m *pipelineManifest) StackName() string {
```

Comment:
to further on the comment above: these methods would belong to a struct in the `cloudformation` pkg

Reply:
Let's discuss